### PR TITLE
Update Chumimi.lua

### PR DESCRIPTION
**Pull request by AaronDemoncia**

### CODE REVIEW

**Comment by xdemolish** [comment]
https://github.com/DarkstarProject/darkstar#pull-requests
amend last commit > change commit message > force push

same with your other pull requests (leave a comment on each once we know what exactly we are merging)

**Comment by Hozu** [comment]
Just going to copy my comment to this one since the other one is closed now.

I'll give you the AF hands quest, but not this one. It makes no sense from a story perspective to have solved the problem by completing AF3, then having the people try to figure out how to solve said problem in AF2 afterwards. Or better yet, being asked to fix some doll that is still locked in Castle Zvahl because you never finished AF2.

Also, you linked the hands quest there. http://ffxi.allakhazam.com/db/quests.html?fquest=276

It's fine as-is.

**Comment by xdemolish** [comment]
squash your commits and force push
also, apparently you haven't changed anything besides a line-ending now

**Comment by xdemolish** [comment]
fix conflicts and drop a comment once it werks

File: scripts/zones/Upper_Jeuno/npcs/Guslam.lua [hunk 1]
```diff
@@ -46,63 +46,63 @@ function onTrigger(player,npc)
...
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_LURKING_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for NIN
 		elseif(player:getMainJob() == 14 and 
-			   player:getQuestStatus(SANDORIA,CHASING_QUOTAS) == QUEST_COMPLETED and 
+			   player:getQuestStatus(SANDORIA,CHASING_QUOTAS) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_DRAGON_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for DRG
 		elseif(player:getMainJob() == 15 and 
-			   player:getQuestStatus(WINDURST,CLASS_REUNION) == QUEST_COMPLETED and 
+			   player:getQuestStatus(WINDURST,CLASS_REUNION) ~= QUEST_AVAILABLE and
```
Comment (TeoTwawki) [inline scripts/zones/Upper_Jeuno/npcs/Guslam.lua]: #1826 "BLM AF3 quest could be started when AF2 quest is accepted. This is not right - AF2 needs to be completed before AF3 can begin, so I added a check for that as well."

Comment (TeoTwawki) [inline scripts/zones/Upper_Jeuno/npcs/Guslam.lua]: Wiki seems to concur with his statement

File: sql/item_mods.sql [hunk 2]
```diff
@@ -29921,81 +29921,81 @@ INSERT INTO `item_mods` VALUES (27725,31,75);
 INSERT INTO `item_mods` VALUES (27725,68,36);
 INSERT INTO `item_mods` VALUES (27725,296,3);
 INSERT INTO `item_mods` VALUES (27725,384,61);
-INSERT INTO `item_mods` VALUES (27728, 1, 111);
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: I think you screwed up your rebase.

**Comment by xdemolish** [comment]
reopen when requested changes have been made (same with your other pull)

**Pull request closed without merging**